src/pages/HomePage.jsx

```diff
@@ -1,13 +1,27 @@
 import React from "react";
 import ChoreList from "../components/ChoreList/ChoreList";
 
-function HomePage({ chores, onShowModal }) {
+function HomePage({ chores, onShowModal, setChores }) {
+  const handleDelete = (id) => {
```

Comment:
Is there a reason you decided to have this and 'handleUpdate' here vs in ChoreUtils?

Reply:
ChoreUtils was supposed to just contain general and reusable helper functions. Keeping handleDelete and handleUpdate there allows me to pass setChores, which is a React state, something I did not want to include in ChoresUtils.